include/arch/arm/aarch32/cortex_m/scripts/linker.ld

```diff
@@ -151,7 +151,7 @@ SECTIONS
 
 #endif /* CONFIG_CODE_DATA_RELOCATION */
 
-    SECTION_PROLOGUE(_TEXT_SECTION_NAME_2,,)
+    SECTION_PROLOGUE(_TEXT_SECTION_NAME,,)
```

Comment:
Should we do the same fix for cortex-r ?

Reply:
Will do that on separate PR, hopefully by the cortex r contributors